mypy/test/teststubgen.py

```diff
@@ -778,6 +780,20 @@ def test(arg0: str) -> None:
         assert_equal(output, ['def test(arg0: str) -> Action: ...'])
         assert_equal(imports, [])
 
+    def test_generate_c_property_with_pybind11(self) -> None:
+        """Signatures included by PyBind11 inside property.fget are read."""
+        class TestClass:
+            def get_attribute(self) -> str:
+                """
+                (self: TestClass) -> str
+                """
+                pass
+            attribute = property(get_attribute, doc="")
```

Comment:
Would it make sense to also have a test case which provides the docstring here as the `doc` argument?

Reply:
So this is not the behavior of PyBind, which only document `fget` `fset` given that they have separate signature.
That being said, `generate_c_property_stub` does check for that docstring (to avoid managing individually too many conditions).
It could be another test, but I believe the question really is what is the expected behavior here?